app/assets/stylesheets/accomodations.scss

```diff
@@ -0,0 +1,14 @@
+// Place all the styles related to the accomodations controller here.
```

Comment:
Find + Replace all instances of `accomodation(s)` with `accommodation(s)` including in the filename :speak_no_evil: